cmd/clusterctl/client/delete.go

```diff
@@ -30,18 +30,18 @@ type DeleteOptions struct {
 	// default rules for kubeconfig discovery will be used.
 	Kubeconfig Kubeconfig
 
-	// CoreProvider version (e.g. cluster-api:v0.3.0) to add to the management cluster. If unspecified, the
+	// CoreProvider version (e.g. cluster-api:v1.1.5) to add to the management cluster. If unspecified, the
```

Comment:
1.2?

Reply:
I went for this as 1.1.5 is released, recent and stable - I don't think we need the most recent versions possible in these examples (but more recent than 0.3.0 would be nice 🙂